src/planner.h

```diff
@@ -64,6 +64,18 @@ class Planner {
   // Precondition: BringPathToHalt() had been called before.
   void SetExternalPosition(GCodeParserAxis axis, float pos);
 
+  // Set the internal planner queue size.
+  // Used to trade-off between command sent - executed motion latency
+  // and maximum achievable speed.
+  // Requires 0 < size <= GetMaxLookahead(), returns true on success.
+  bool SetLookahead(int size);
```

Comment:
Now that the planner has some way to set the queue size: are there tests we want in the planner_test.cc as well ? e.g. figure out how the acceleration properties change when we change the queue size ?